### PR TITLE
Add new feature: New Button & Input to send chat commands while open inventory

### PR DESCRIPTION
**Pull request by alexandrosmagos**
As the title suggests, something that is long overdue and would really help in dupe hunting, the ability to send chat commands while having an inventory open.

image

### CODE REVIEW

**Comment by Pxddyk45** [comment]
The variable `screenWidth` is defined nowhere

File: src/main/java/org/uiutils/mixin/HandledScreenMixin.java [hunk 1]
```diff
@@ -378,6 +381,29 @@ public void init(CallbackInfo ci) {
                     e.printStackTrace();
                 }
             }).width(120).position(5, 215).build());
+
```
Comment (Pxddyk45) [inline src/main/java/org/uiutils/mixin/HandledScreenMixin.java:384]: add `int screenWidth = MinecraftClient.getInstance().currentScreen.width;` here

**Comment by Pxddyk45** [comment]
Also, this disables Q for drop in inventory

File: src/main/java/org/uiutils/mixin/HandledScreenMixin.java (hunk 1 above)
Comment (alexandrosmagos) [reply src/main/java/org/uiutils/mixin/HandledScreenMixin.java:385]: Forgot to add it to the initial commit. Thanks

**Comment by alexandrosmagos** [comment]
> Also, this disables Q for drop in inventory

Corrected as well, thanks again.

**Comment by Pxddyk45** [comment]
Almost looks good, only F for offhand and 1-9 for hotbar don't work

**Comment by alexandrosmagos** [comment]
Added those now too. :)

**Comment by Pxddyk45** [comment]
Now it only works in inventory, not in for example chest screens

**Comment by Pxddyk45** [comment]
and you cant type E because it exits out of the textbox and pressing 1-9 selects the slot in the hotbar instead of moving it to 1-9

**Comment by alexandrosmagos** [comment]
> Now it only works in inventory, not in for example chest screens

Mine works on all screens, 🤔



> and you cant type E because it exits out of the textbox and pressing 1-9 selects the slot in the hotbar instead of moving it to 1-9

Typing e while the input is focused, doesn't exit in all screens except the inventory screen. Will fix it later

**Comment by Pxddyk45** [comment]
> > Now it only works in inventory, not in for example chest screens
> 
> Mine works on all screens, 🤔
> 
I don't remember what I meant was not working but didn't mean the chat sender

**Comment by Coderx-Gamer** [comment]
This got implemented before this was merged and there are conflicts, closing.

**Pull request closed without merging**